crates/forge/tests/cli/compiler.rs

```diff
@@ -94,7 +94,7 @@ forgetest!(can_list_resolved_compiler_versions_verbose, |prj, cmd| {
     cmd.args(["compiler", "resolve", "-v"]).assert_success().stdout_eq(str![[r#"
 Solidity:
 
-0.8.27:
+0.8.27 [Max supported EVM version: [..]]:
```

Comment:
Perhaps we can use a shorter syntax like `(<= london)`and move this to verbosity mode `-vv`